lib/websocket/websocket-handler.ts

```diff
@@ -121,10 +121,18 @@ function setup(socket: Interface.ISocket): void
 
                 // Emit the data
                 // TODO: Acknowledgement callback function?
-                socket.send('data', {'correlationId': sub.correlationId,
-                                     'data': data});
+                socket.send('data', {
+                    'correlationId': sub.correlationId,
+                    'data': data
+                });
                 socket.log.info('Data sent');
             });
+
+            sub.on('error', (err: Error): void => {
+                sub.removeAllListeners();
```

Comment:
The ordering differs between here and the other file. Intentional or unimportant?